src/main/java/org/example/bookstore/model/Book.java

```diff
@@ -0,0 +1,29 @@
+package org.example.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
`@Data` is not recommended to use with `@Entity` because of possible poor performance